chunky/src/java/se/llbit/chunky/ui/controller/ChunkyFxController.java

```diff
@@ -162,17 +162,23 @@ public class ChunkyFxController
     @Override public void setProgress(String task, int done, int start, int target) {
       Platform.runLater(() -> {
         progressBar.setProgress((double) done / (target - start));
-        progressLbl.setText(String.format("%s: %s of %s", task, decimalFormat.format(done),
-            decimalFormat.format(target)));
+        if (target - start > 0) {
+          progressLbl.setText(String.format("%s - %s%%", task, Math.round(100 * (double) done / (target - start))));
```

Comment:
`%s` is for strings, maybe we can even have one decimal digit (when progress is slow):

```suggestion
          progressLbl.setText(String.format("%s - %.1f%%", task, Math.round(100 * (double) done / (target - start))));
```

Reply:
Yeah, `%f` makes more sense than `%s`.

Personally, I found that adding a decimal digit gave a bit too much visual weight to the element. Also, the progress bar and SPP counter should already give some indication of progress. I'll experiment with it again, though.

Reply:
Adding replacing the hyphen with an en dash fixes the visual weight issue, so I've gone ahead and added the extra digit.